### PR TITLE
Update CurlRestClient.php

### PR DESCRIPTION
**Pull request by insekticid**
added setOptions()

$curl = new CurlRestClient();
$curl->setOptions([CURLOPT_SSL_VERIFYHOST => 0, CURLOPT_SSL_VERIFYPEER => 0]);

### CODE REVIEW

**Comment by Atriedes** [comment]
could you please also update readme.md ?

**Comment by insekticid** [comment]
updated

File: README.md [hunk 1]
```diff
@@ -31,6 +32,7 @@ include 'vendor/autoload.php';
 use RestClient\CurlRestClient;
 
 $curl = new CurlRestClient();
+$curl->setOptions([CURLOPT_SSL_VERIFYHOST => 0, CURLOPT_SSL_VERIFYPEER => 0]);
```
Comment (delmicio) [inline README.md:35]: This is not PHP 5.3 compatible

Comment (insekticid) [reply README.md:35]: PHP 5.3 got EOL

Comment (delmicio) [reply README.md:35]: I know, but this library is 5.3+ compatible

> Simple cURL PHP Rest Client library for PHP 5.3+

**Pull request merged**